solvers/stereo_imaging/time_window_pruned_stereo_milp/scripts/validate_matrix.py

```
@@ -0,0 +1,262 @@
+
+    status_path = solution_dir / "status.json"
+    if not status_path.exists():
+        return {"error": "status.json missing", "elapsed": elapsed}
+
+    status = json.loads(status_path.read_text(encoding="utf-8"))
+    status["elapsed"] = elapsed
+    return status
+
+
+def _run_verifier(case_dir: Path, solution_path: Path) -> dict[str, Any]:
+    proc = subprocess.run(
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)